Formula/numpy.rb

```diff
@@ -19,6 +19,13 @@ class Numpy < Formula
   depends_on "openblas"
   depends_on "python@3.9"
 
+  # Needed on Linux to force gcc 10 for AVX512 support
+  fails_with gcc: "5"
+  fails_with gcc: "6"
+  fails_with gcc: "7"
+  fails_with gcc: "8"
+  fails_with gcc: "9"
```

Comment:
I thought this was fixed now so that
```ruby
fails_with gcc: "9"
```
is enough to exclude all `gcc`s older than `gcc-9`?

Reply:
@iMichka can you confirm that?  I lost track of where we were on this.  Ideally this should be the right way to do this!

Reply:
`fails_with gcc: "5"` is enough. This is what I did almost everywhere.

But I don't understand the fails_with logic and I do not agree with it, so I do not wish to engage anymore on that topic as it leads to endless bikeshedding.

Reply:
Okay, I'm just doing `fails_with gcc: "5"`.  I think the main source of confusion has been over what is necessary to build on a CI environment versus accounting for other scenarios when users build from source locally.  Since I'm hoping we can eventually eliminate (or at least greatly reduce) the need to build from source by end users altogether, this may end up being more of a niche concern at that point.